tests/Unit/API/Management/JobsTest.php

```diff
@@ -28,7 +28,7 @@
     fwrite($import, '[{"email":"php-sdk-test-import-user-job@auth0.com","email_verified":true,"app_metadata":{"roles":["admin","super"],"plan":"premium"},"user_metadata":{"theme":"dark"}}]');
 
     $path = stream_get_meta_data($import)['uri'];
-    $keyOffset = 3;
+    $keyOffset = 2;
```

Comment:
This fix is due to a change in php-http/multipart-stream-builder.

In v1.3.1, body has a `Content-Length` header (as shown below).

```
--66e64e8fe1f375.28637093
Content-Disposition: form-data; name="users”
Content-Length: 167

[{“email”: “php-sdk-test-import-user-job@auth0.com”, “email_verified”:true, “app_metadata”:{“roles”:[“admin”, “super”], “plan”: “premium “}, “user_metadata”:{“theme”: “dark”}}]
--66e64e8fe1f375.28637093
Content-Disposition: form-data; name="connection_id”
Content-Length: 16

__test_conn_id__
--66e64e8fe1f375.28637093
Content-Disposition: form-data; name="upsert”
Content-Length: 4

Content-Disposition: form-data; name=“upsert” Content-Length: 4
...
```

However, [after v.1.3.1, the `Content-Length` header has been removed](https://github.com/php-http/multipart-stream-builder/blob/1.x/CHANGELOG.md#140---2024-09-01), so `$keyOffset` is misaligned and the test fails.

Therefore, we think it is better to fix the test with this fix to follow the changes in multipart-stream-builder.

Reply:
(And I think it is better to manage lock files by git as well.)